apm-sniffer/config/agent.config

```diff
@@ -320,3 +320,15 @@ plugin.nettyhttp.supported_content_types_prefix=${SW_PLUGIN_NETTYHTTP_SUPPORTED_
 plugin.rocketmqclient.collect_message_keys=${SW_PLUGIN_ROCKETMQCLIENT_COLLECT_MESSAGE_KEYS:false}
 # If set to true, the tags of messages would be collected by the plugin for RocketMQ Java client.
 plugin.rocketmqclient.collect_message_tags=${SW_PLUGIN_ROCKETMQCLIENT_COLLECT_MESSAGE_TAGS:false}
+# How many parameter characters to keep and send to the OAP backend, -1 save all, 0 save nothing, >0 save specified length of parameter string to the tag. default is 0.
```

Comment:
```suggestion
# Define the max length of collected HTTP parameters. The default value(=0) means not collecting.
```

Reply:
Please remove `-1`, we can't afford to collect all and make agent impact the target memory cost unpredictable.

Reply:
Meanwhile, the length is recommended to use to control boday and parameters. Headers should be limited by key list.

Reply:
Okay, no problem.